Octokit/Models/Request/BranchProtectionUpdate.cs

```diff
@@ -29,15 +29,32 @@ public BranchProtectionSettingsUpdate(BranchProtectionRequiredStatusChecksUpdate
         /// <summary>
         /// Create a BranchProtection update request
         /// </summary>
+        /// <param name="restrictions">Specifies the requested push access restrictions (applies only to Organization owned repositories). Pass null to disable push access restrictions</param>
+        public BranchProtectionSettingsUpdate(BranchProtectionPushRestrictionsUpdate restrictions)
+        {
+            RequiredStatusChecks = null;
+            Restrictions = restrictions;
+        }
+
```

Comment:
Should there be a `ctor` taking just the `EnforceAdmins` bool (and setting the other 2 class members to `null`) ?